easypy/timing.py

```diff
@@ -201,7 +202,19 @@ def pred():
 
 
 def iter_wait(timeout, pred=None, sleep=0.5, message=None,
-              progressbar=True, throw=True, allow_interruption=False, caption=None):
+              progressbar=True, throw=True, allow_interruption=False, caption=None, from_wait=False):
+
+    # Calling wait() or iter_wait() with a predicate and no message is very
+    # not informative and is pending deprecation
+    if pred and not message:
+        func = 'wait' if from_wait else 'iter_wait'
+        stack_level = 3 if from_wait else 2
```

Comment:
_There is no way to add a message to repeat, so it'll always print a warning._
If `throw` is False the no warning will be raised.\
@koreno